ejp-rd-vp/vp-dp-frontend
# 138 unused filters shouldnt be sent to beacon services

**What's in the PR**
*  Send only the search parameters selected by the user during the search request.
*  The search filters should be deselected by default.

**How to test manually**
* Go to Resource Discovery
* Check the search filters if they are not selected by default.
* select some filters and execute a search
* Check in the console that only the selected search filters are sent to the backend.
